src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -150,12 +165,15 @@ private void AnalysisConfigMonitor_ConfigChanged(object sender, EventArgs e)
 
         private async Task StopServer()
         {
+            await threadHandling.SwitchToBackgroundThread();
```

Comment:
I can add a test that verifies that the event switches to a background thread, but I'm not sure how much value it will have.

Reply:
I think it's worth adding, particularly if we can check the order of invocations i.e. switch to background thread is called before the close.

Reply:
Actually, let's commit without the test so we can get a final product version to build.